jetty-util/src/main/java/org/eclipse/jetty/util/MultiPartInputStreamParser.java

```diff
@@ -934,14 +935,14 @@ private String filenameValue(String nameEqualsValue)
     }
 
 
-
+    // TODO: considers switching to Base64.getMimeDecoder().wrap(InputStream)
     private static class Base64InputStream extends InputStream
```

Comment:
This could be a useful change in `MultiPartInputStreamParser`, using the `java.util.Base64.Encoder#wrap(InputStream)` instead of this class.
@lachlan-roberts what do you think?